install/third-party/ext-hcp_vault/install.yml

```diff
@@ -0,0 +1,14 @@
+id: ext-hcp_vault
```

Comment:
```suggestion
id: ext-hcp-vault
```

Reply:
I've performed the suggested changes, thanks